flytekit/types/schema/types_pandas.py

```diff
@@ -10,14 +10,17 @@
 from flytekit.plugins import pandas
```

Comment:
@kumare3 can you take a look at this one line please?  We should just swap it out right?

Reply:
yup, flytekit has a direct dependency on pandas.